src/main/java/com/post/menagment/services/PostOfficeServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.post.menagment.services;
+
+import com.post.menagment.model.PostOffice;
+import com.post.menagment.repository.PostOfficeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PostOfficeServiceImpl implements PostOfficeService{
+
+    private PostOfficeRepository postOfficeRepository;
+
+    @Autowired
+    public PostOfficeServiceImpl(PostOfficeRepository postOfficeRepository) {
+        this.postOfficeRepository = postOfficeRepository;
+    }
+
+    @Override
+    public boolean isPostOfficeAvailable(long postOfficeId) {
```

Comment:
You forgot to change type for argument to Long and proceed null value. With such implementation there is not any profit from update in controller.

Reply:
done